src/components/landing_page/Event.js

```diff
@@ -12,20 +12,20 @@ function EventTab(props){
     if (props.index === 0){
         return(
             <div>
-                <h1 className={classes.past1}>Winter 2020 Mentorship Program</h1>
+                <h1 className={classes.past1}>Spring 2020 Coffee Chat Program</h1>
                 <p className={classes.past1_desc}>
-                    A description of the project 
-                    <Button className={classes.past1_button}>Learn More</Button>
+                    Coffee Chats are an opportunity for students to connect with multiple experienced upper year students to explore more areas of technology. (107 Coffee Chat pairs)
+                    {/* <Button className={classes.past1_button}>Learn More</Button> */}
                 </p>
-                <h1 className={classes.past1}>Spring 2020 Coffee Chat Program</h1>
+                <h1 className={classes.past1}>Fall 2020 Mentorship Program</h1>
                 <p className={classes.past1_desc}>
-                    A description of the project 
-                    <Button className={classes.past1_button}>Learn More</Button>
+                    Mentorship program is an ongoing program to pair first year with upper year mentors to provide guidance. (110 Mentor/Mentee pairs)  
```

Comment:
```suggestion
                    Mentorship is an ongoing program that pairs lower year students with upper year mentors to provide guidance. (110 Mentor/Mentee pairs)  
```
is the 110 pairs in total or just for this term?